pages/index.tsx

```diff
@@ -1,15 +1,9 @@
-import Head from "next/head";
 import Logo from "../components/welcome/Welcome";
 import Link from "next/link";
 
 export default function Home() {
   return (
     <>
-      <Head>
-        <title>Meet Albert</title>
-        <link rel="icon" href="/albert-hey.ico" />
```

Comment:
It would be better if your favicon would be called favicon.ico, so everybody knows this is your favicon file.